http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpSamDAO.scala

```diff
@@ -223,6 +254,40 @@ class HttpSamDAO[F[_]](httpClient: Client[F],
         }
     } yield ()
 
+  override def createResourceWithParentV2[R](resource: R,
+                                             creatorEmail: WorkbenchEmail,
+                                             cloudContext: CloudContext,
+                                             workspaceId: WorkspaceId,
+                                             userInfo: UserInfo
+  )(implicit sr: SamResource[R], encoder: Encoder[R], ev: Ask[F, TraceId]): F[Unit] = for {
+    traceId <- ev.ask
+    authHeader = Authorization(Credentials.Token(AuthScheme.Bearer, userInfo.accessToken.token))
+    _ <- logger.info(
+      s"${traceId} | creating ${sr.resourceType.asString} resource in sam v2 for ${cloudContext}/${sr.resourceIdAsString(resource)}"
+    )
+    _ <- metrics.incrementCounter(s"sam/createResource/${sr.resourceType.asString}")
+    policies = Map[SamPolicyName, SamPolicyData](
+      SamPolicyName.Creator -> SamPolicyData(List(creatorEmail), List(SamRole.Creator))
+    )
+    parent = SerializableSamResource(SamResourceType.Project, WorkspaceResourceSamResourceId(workspaceId))
```

Comment:
I don't think this is right -- it's using `Project` as the resource type and the workspace id. I think the best thing to do for now is continue to use `SamResourceType.Project` and `ProjectSamResourceId(googleProject)` if the cloudContext is GCP -- and just not set a parent if the cloudContext is Azure. 

(We lose the inherited project-owner permissions in that case -- but honestly we'll be revisiting the Sam stuff anyway when we implement multi-tenancy.)